main/lsp/LSPTask.cc

```diff
@@ -54,7 +54,7 @@ ResponseMessageStatus statusForResponse(const ResponseMessage &response) {
                     // textDocument/documentSymbol
                     return ResponseMessageStatus::Unknown;
                 } else if constexpr (is_same_v<T, variant<JSONNullObject, vector<unique_ptr<Location>>>>) {
-                    // textDocument/definition, textDocument/typeDefinition, textDocument/references, and
+                    // textDocument/definition, textDocument/typeDefinition, textDocument/references and
```

Comment:
You don't like Oxford commas?

Reply:
Complete accident, but come to think of it I'm not a fan lol. 

Will revert.